### PR TITLE
feat(meetings/meeting): add size-only layout requests

### PR DESCRIPTION
**Pull request by InteractiveTimmy**
# Description

The scope of the changes in this pull request are to add the ability to change the layout size without forcing a layout type.

## Links

[JIRA](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-370386).

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 1]
```diff
@@ -5182,9 +5182,6 @@ export default class Meeting extends StatelessWebexPlugin {
     if (LAYOUT_TYPES.includes(layoutType)) {
       layoutInfo.layoutType = layoutType;
     }
-    else {
```
Comment (marcin-bazyl) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:5185]: I think we should still fail if a non-undefined layoutType is passed in that doesn't match any valid value. We should maybe also update the docs for this function to say that layoutType is optional?

File: packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js [hunk 2]
```diff
@@ -1942,9 +1942,19 @@ describe('plugin-meetings', () => {
             });
           });
 
-          it('doesn\'t have layoutType which exists in the list of allowed layoutTypes should throw an error', async () => {
+          it('doesn\'t have layoutType which exists in the list of allowed layoutTypes should not assign for payload', async () => {
```
Comment (marcin-bazyl) [inline packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js:1945]: I think we should leave that test as it was as we don't want to just silently ignore "invalid layout" and instead add a new test that sets the layout to some value and then calls changeVideoLayout with layoutType being undefined and verify that we send the previous value to the server.

Comment (marcin-bazyl) [reply packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js:1945]: might be also worth testing that the scenario where changeVideoLayout is called first time on a meeting object without layoutType - in that case we should use undefined value for it as there is no previous value available

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js (hunk 1 above)
Comment (InteractiveTimmy) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:5185]: I've updated the source to address this. layoutType is now optional in JSDoc. This conditional needs to stay due to a previously existing layoutInfo validation check later within this function.

File: packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js (hunk 2 above)
Comment (marcin-bazyl) [reply packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js:1945]: but wouldn't it be useful to have a test that checks the following:
if some non-undefined, valid layout is set and changeLayout() is called without a layout type, then verify that we send a request without layout being specified

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js (hunk 1 above)
Comment (marcin-bazyl) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:5185]: ok, but now we silently ignore any invalid `layoutType` values?

Comment (InteractiveTimmy) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:5185]: I believe I understand now. I've made some changes, please re-review.

File: packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js (hunk 2 above)
Comment (InteractiveTimmy) [reply packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js]: I believe I understand now. I've made some changes, please re-review.

**Review by marcin-bazyl — APPROVED** [review]

**Pull request merged**